content/4_Gold/Sliding.mdx

```diff
@@ -38,10 +38,18 @@ export const problems = {
 			'1076', 
 			'Normal', 
 			false, 
-			['Set'],
+			['Sliding Window', 'Set'],
 			'cses-1076'
 		),
-		new Problem('CSES', 'Sliding Cost', '1077', 'Hard', false, [], ''),
+		new Problem(
+			'CSES', 
+			'Sliding Cost', 
+			'1077', 
+			'Hard', 
+			false, 
+			['Sliding Window', 'PURS'], 
```

Comment:
You don't need PURS. The cost is just the max K/2 numbers in the window minus the min K/2.

Reply:
Oh cool, I'll change up the solution to reflect this